src/file_manager/file_manager.py

```
@@ -25,12 +30,13 @@ def read_content(self):
 
         It returns a string containing the content read
         """
-        file_content = ''
+        file_content = ''        
         try:
             fo = open(self.file_path)
             file_content = fo.read()
         except IOError:
-            raise IOError('cannot read file')
+            logger.error("cannot read file ")
+            raise IOError('cannot read file')            
```

since we are logging the error we wont need to raise the error again